tests/unit/test_workflow_wiring.py

```
@@ -500,9 +500,34 @@ def test_publish_npm_exposes_dist_tag_for_backfills() -> None:
+_LGTM_CI_PIN = "66cad82ead0e5d119928c895c7d7da9c837989e5"
+
+
+def test_all_lgtm_ci_refs_use_the_canonical_pin() -> None:
+    """Every lgtm-ci ref in workflows must match the single canonical pin.
+
+    Guards the repo-wide invariant from #1280: `uses:` refs and
+    `tooling-ref:` inputs all point at the same lgtm-ci commit, so pins
+    cannot silently drift apart again.
+    """
+    ref_pattern = re.compile(
+        r"lgtm-hq/lgtm-ci/[^@\s]+@([0-9a-f]{40})" r"|tooling-ref: '([0-9a-f]{40})'",
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)